### ISSUE TITLE
@see {@link ...} seems to be broken - Error reported

### ISSUE THREAD

**Issue by Alaneor** [hegemonic/jsdoc-baseline#149]
The following tag in my jsdoc

`@see       {@link http://cs.wikipedia.org/wiki/Akademický_titul Wikipedia}`

Throws the following error while generating docs:

```
[Error: unable to parse <a href="http://cs.wikipedia.org/wiki/Akademick%C3%BD_titul">Wikipedia</a>: Expected "!", "$", "'", "(", "*", "...", "?", "Function", "\"", "\\", "_", "break", "case", "catch", "class", "const", "continue", "debugger", "default", "delete", "do", "else", "enum", "export", "extends", "false", "finally", "for", "function", "if", "implements", "import", "in", "instanceof", "interface", "let", "new", "null", "package", "private", "protected", "public", "return", "static", "super", "switch", "this", "throw", "true", "try", "typeof", "undefined", "var", "void", "while", "with", "yield", "{", Unicode letter number, Unicode lowercase letter, Unicode modifier letter, Unicode other letter, Unicode titlecase letter or Unicode uppercase letter but "<" found.]
```

Removing the {@link} tag and only using the URL allows the docs to be built without errors, but then I cannot alias the link into something meaningful for the reader.

Thank you for your assistance in this matter! Let me know if I can provide further troubleshooting information.

**Comment by hegemonic** [comment]
Thanks for the heads up! This issue was caused by a bug in JSDoc, so I've committed a fix to the JSDoc repo. The fix will be included in JSDoc 3.3.0. Until then, you can use the latest version from the master or releases/3.3 branches.

**Comment by Alaneor** [comment]
Ah, great! Thank you for your help!
